cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1374,15 +1374,6 @@ def test_advanced_components(self):
         self.assertNotEqual(only_template.get('category'), 'video')
         self.assertNotEqual(only_template.get('category'), 'openassessment')
 
-    def test_advanced_components_without_display_name(self):
```

Comment:
@dianakhuang do you have any thoughts here? I really think this test is unfortunately a by product of the crap way GST is defined.